libraries/jevmodal/media/css/jevmodal.css

```diff
@@ -176,6 +176,9 @@
     border:none;
 }
 
+/** Hide the modal body by default **/
+.jevbootstrap > div.action_dialogJQ  {display:none;}
```

Comment:
Was it visible on some sites before its opened?
I'd like to make sure this is the best fix for the problem.  Note I found an issue with modals not closing on sites with MooTools still loaded when RSVP Pro was loaded too in event detail page - that was fixed in 3.4.9